app/views/non_hmis_assessments/_common_questions.haml

```diff
@@ -0,0 +1,122 @@
+.c-question.c-question--primary
+  .c-question__wrapper
+    .c-question__number
+    - label = 'Date of Assessment'
+    - if disabled
+      = render 'non_hmis_assessments/question_display', value: @assessment.entry_date, label: label
+    - else
+      .c-question__input
+        = form.input :entry_date, label: label, as: :date_picker
+.c-question
+  = form.input :domestic_violence, as: :pretty_boolean, wrapper: :custom_boolean, pre_label: '2B.', label: 'Are you currently fleeing or attempting to flee a domestic violence situation?', hint: 'Examples: sexual assault, stalking, domestic violence'
+.c-question
+  .c-question__wrapper
+    .c-question__number 4D.
+    - label = _('3D. Total # of Boston Homeless Nights')
+    - if disabled
+      = render 'non_hmis_assessments/question_display', value: @assessment.days_homeless_in_the_last_three_years, label: label
```

Comment:
Do we actually use this file?  It feels like this should be covered by `@assessment.class.form_fields`